systems/plants/@RigidBodyManipulator/geometricJacobian.m

```diff
@@ -1,30 +1,36 @@
-function [J, v_indices, dJ] = geometricJacobian(obj, kinsol, base, end_effector, expressed_in)
-%GEOMETRICJACOBIAN Computes the geometric Jacobian from base to endEffector
+function [J, v_or_qdot_indices, dJ] = geometricJacobian(obj, kinsol, base, end_effector, expressed_in, in_terms_of_qdot)
+%GEOMETRICJACOBIAN Computes the geometric Jacobian from base to end_effector
```

Comment:
I think I actually prefer it as it is for now, as I don't see any other options being added in the near future, and the expressed_in and in_terms_of_qdot arguments are often needed.